src/pages/Details.js

```diff
@@ -2,83 +2,82 @@ import React from 'react';
-  //let var = this.props.location.state;
+ 
+     });
+ }
+ //console.log({this.props.location.state})
+ //let currentSubmission;
+ //var currentSubmission = props.location.state.mySubmission;
+ //let var = this.props.location.state;
+  render() {
+   console.log("hello");
+   console.log(this.props.location);
+   return (
+ 
+   <React.Fragment>
+      <Carousel>
+       <Carousel.Item>
+         <img src={this.props.location.submission.images[0]}  alt="logo"/>
+       </Carousel.Item>
+       <Carousel.Item>
+         <img src={logo}  alt="logo"/> 
+       </Carousel.Item>
+       <Carousel.Item>
+         <img src={logo}  alt="logo"/>
+       </Carousel.Item>
+     </Carousel>
+ 
+     <div className="card-body">
+       <h1 className="title-event">{}</h1>
+ 
+       {this.props.location.submission.subjectName && <h2 className="title-location"> {this.props.location.submission.subjectName}</h2>}
+      
+     
+       <br></br>
   
-	render() {
-    console.log("hello");
-    console.log(this.props.location);
-		return (
-
-    <React.Fragment> 
-       <Carousel>
-        <Carousel.Item>
-          <img src={this.props.location.submission.images[0]}  alt="logo"/>
-        </Carousel.Item>
-        <Carousel.Item>
-          <img src={logo}  alt="logo"/>  
-        </Carousel.Item>
-        <Carousel.Item>
-          <img src={logo}  alt="logo"/>
-        </Carousel.Item>
-      </Carousel>
-
-      <div className="card-body">
-        <h1 className="title-event">{}</h1>
-
-        {this.props.location.submission.subjectName && <h2 className="title-location"> {this.props.location.submission.subjectName}</h2>}
-
-        <br></br>
-    
-
-
-        {/* {this.props.location.submission.submitDate && <h2 className="title-date"> By {this.props.location.submission.studentName}</h2>}  */}
-        <div className="row=title">
-
-
-        {this.props.location.submission.location && <h3 className="submitter-location">  {this.props.location.submission.location}</h3>}
-        {dateToString(this.props.location.submission.eventDate) && <h3 className="submitter-location">  {dateToString(this.props.location.submission.eventDate)} </h3>}
-
-
-        {this.props.location.submission.description && <p> {this.props.location.submission.description}</p>}
-
-        </div>
-
-        <br></br>
-
-
-      </div>
+ 
+ 
+       {/* {this.props.location.submission.submitDate && <h2 className="title-date"> By {this.props.location.submission.studentName}</h2>}  */}
+       <div className="row=title">
+ 
+ 
+       {this.props.location.submission.location && <h3 className="submitter-location">  {this.props.location.submission.location}</h3>}
+       {dateToString(this.props.location.submission.eventDate) && <h3 className="submitter-location">  {dateToString(this.props.location.submission.eventDate)} </h3>}
+ 
+ 
+       {this.props.location.submission.description && <p> {this.props.location.submission.description}</p>}
+ 
+       </div>
+ 
+       <br></br>
+ 
+ 
```

Comment:
this is super nitpick but remove extra space lol the spacing is kinda inconsistent in general

Reply:
choose between 1 new line vs 2 new line with spacing? keep consistent